app/views/main.scala.html

```diff
@@ -15,7 +15,7 @@
             padding-top: 72px;
         }
     </style>
-    <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("build/main.css")"/>
+    <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("build/app.css")"/>
```

Comment:
Why the change?

Reply:
This splits out the main and admin css into different files, whereas previously it did not.